src/gen_rpc_acceptor.erl

```diff
@@ -88,6 +88,12 @@ waiting_for_data({data, Data}, #state{socket=Socket,client_node=Node} = State) -
     %% The meat of the whole project: process a function call and return
     %% the data
     try erlang:binary_to_term(Data) of
+        {Node, ClientPid, Ref, {block_call, M, F, A}} ->
+            WorkerPid = erlang:spawn(?MODULE, call_worker, [self(), ClientPid, Ref, M, F, A]),
```

Comment:
Isn't this a "non-blocking" call? The server is supposed to run the function inside the body and reply with the results instead of spawning a worker.